gobblin-data-management/src/main/java/org/apache/gobblin/data/management/copy/CopyableFile.java

```diff
@@ -147,6 +148,36 @@ public static Builder fromOriginAndDestination(FileSystem originFs, FileStatus o
         .preserve(copyConfiguration.getPreserve());
   }
 
+
+  /**
+   * Set file system based source and destination dataset of a {@link CopyableFile}
+   *
+   * @param copyableFile {@link CopyableFile} in action
+   * @param originFs {@link FileSystem} where the {@code copyableFile} exists
+   * @param copyConfiguration {@link CopyConfiguration} for the copy job
+   */
+  public static void setFsDatasets(CopyableFile copyableFile, FileSystem originFs, CopyConfiguration copyConfiguration) {
```

Comment:
This looks like it could be a member method since it is manipulating copyableFile.